src/client/pages/create.js

```diff
@@ -220,6 +240,37 @@ class Create extends React.Component {
               </Col>
             </FormGroup>
             <hr />
+            <FormGroup row>
+              <Label for="turnDuration" sm={2}>Turn Duration</Label>
+              <Col sm={10}>
+                <Input type="select" name="turnDuration" id="turnDuration" onChange={e => this.onTurnDurationUpdated(e)} value={this.state.turnDuration}>
+                  <option value={0}>No Timer</option>
```

Comment:
I think we should keep it simple. We only need the values
* No Timer
* 3min
* 5min (default)
* 10min

And I would remove the `custom` logic.

Reply:
Oh yeah, I did mean to remove custom. Just forgot